+adi/+internal/ADAR100x.m

```diff
@@ -994,7 +994,7 @@ function LatchTxSettings(obj)
         end
         
         function result = get.Temp(obj)
-            result = zeros(numel(obj.deviceNames), 1);
+            result = zeros(numel(obj.SubarrayToChipMap), 1);
```

Comment:
The dimension of Temp isn't really helpful. You can't tell which chip is the hot one. This should match the other device attribute oriented attributes in dimension.